content/docs/command-reference/plots/diff.md

```diff
@@ -47,14 +47,17 @@ please see `dvc plots`.
 
 - `--targets <path>` - specific metric files to visualize. These must be listed
   in a [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file) file
-  (see the `--plots` option of `dvc run`). When specifying multiple `--targets`
-  and `revisions`, you may use `--` after this option's arguments, e.g.:
+  (see the `--plots` option of `dvc run`). When specifying arguments for
+  `--targets` before `revisions`, you may use `--` after this option's arguments
+  , e.g.:
```

Comment:
@jorgeorpinel `--` will be in only when `--targets` precede `revisions`. Also even if only one `--targets` file is specified, `--` will have to be used. I think current reason is ambiguous so have made some changes here.